roles/container_runtime/tasks/docker_storage_setup_overlay.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Setup the docker-storage for overlay
+  template:
+    src: overlay2_dss.j2
+    dest: /etc/sysconfig/docker-storage-setup
+    owner: root
+    group: root
+    mode: 0664
+  when:
+  - container_runtime_run_docker_storage_setup | default(False)
```

Comment:
get rid of the bool, let's check the variable that sets to storage type.

Reply:
This doesn't really give us a mechanism to turn on or off the storage setup, especially if we default the variable to something.

`container_runtime_docker_storage_setup_type=overlay` 

Should we not specify the variable and then it will not be defined?  Or default it to `''`?